handlers/tracing.go

```diff
@@ -415,3 +417,48 @@ func readQuery(conf *config.Config, values url.Values) (models.TracingQuery, err
 
 	return q, nil
 }
+
+// TracingDiagnose is the API handler to diagnose Tracing configuration
+func TracingDiagnose(
+	conf *config.Config,
+	kialiCache cache.KialiCache,
+	clientFactory kubernetes.ClientFactory,
+	prom prometheus.ClientInterface,
+	cpm business.ControlPlaneMonitor,
+	traceClientLoader func() tracing.ClientInterface,
+	grafana *grafana.Service,
+	discovery *istio.Discovery,
+) http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		// prepare the logger in a context, and replace the request context with ours that has our logger in it
+		r = log.AddGroupToLoggerInRequestContext(r, log.TracingLogName)
+
+		business, err := getLayer(r, conf, kialiCache, clientFactory, cpm, prom, traceClientLoader, grafana, discovery)
+		if err != nil {
+			RespondWithError(w, http.StatusInternalServerError, "Services initialization error: "+err.Error())
+			return
+		}
+		if !isHomeCPAccessible(r.Context(), business.Namespace, clientFactory.GetSAHomeClusterClient().ClusterInfo().Name) {
+			RespondWithError(w, http.StatusInternalServerError, "Services initialization error: "+err.Error())
+			return
+		}
+
+		status, err := business.Tracing.TracingDiagnose(r.Context(), clientFactory.GetSAHomeClusterClient().GetToken())
+		if err != nil {
+			RespondWithError(w, http.StatusServiceUnavailable, err.Error())
+			return
+		}
+
+		RespondWithJSON(w, http.StatusOK, status)
+	}
+}
+
+// Check access to the home istio namespace
+func isHomeCPAccessible(ctx context.Context, namespaceService business.NamespaceService, cluster string) bool {
+	conf := config.Get()
+	_, err := namespaceService.GetClusterNamespace(ctx, conf.IstioNamespace, cluster)
+	if err == nil || !errors.IsForbidden(err) {
```

Comment:
If there's an error other than `errors.IsForbidden` we don't know if the home control plane is accessible or not.

Reply:
I was trying to follow the same approach than here: https://github.com/kiali/kiali/blob/master/handlers/mesh.go#L91
Maybe it should check just that err is nil?

Reply:
> Maybe it should check just that err is nil?

Yes I think so. Otherwise if there's some kind of transient error, network issue or something, and the user is not allowed to see the namespace, this check would still pass.

Reply:
I've pushed the changes this morning but for some reason is not updated... 
![image](https://github.com/user-attachments/assets/1c1d8254-9249-42f8-ba92-0b603d8ee907)